products/coder.md

```diff
@@ -22,6 +22,13 @@ auto:
 # eoas(x) = releaseDate(x+2)
 # eol(x) = releaseDate(x+3)
 releases:
+-   releaseCycle: "2.14"
+    releaseDate: 2024-08-06
+    eoas: false # releaseDate 2.16
+    eol: false # releaseDate 2.17
+    latest: "2.14.1"
+    latestReleaseDate: 024-08-07
```

Comment:
date is incorrectly formatted. https://github.com/coder/coder/releases/tag/v2.14.0 and https://coder.com/docs/install/releases both say 6th August, not 7th.

```suggestion
    latestReleaseDate: 2024-08-06
```

Reply:
2024-08-07 comes from https://github.com/coder/coder/releases/tag/v2.14.1